packages/shorebird_cli/lib/src/executables/aot_tools.dart

```diff
@@ -82,8 +83,13 @@ class AotTools {
     return !_preLinkerFlutterRevisions.contains(flutterRevision);
   }
 
-  Future<ShorebirdProcessResult> _exec(
+  Future<ShorebirdProcessResult> _prepareAndRun(
```

Comment:
I'm not sure the rename adds anything? Might be better to revert the name and add some comments explaining what exec does.

Reply:
It is not really a rename, `_exec` still exits, but the git diff makes it looks like a rename. I will try to rework the names and the nesting to make this easier to read